packages/docsite/scripts/generate-llm-page.sh

```
@@ -0,0 +1,43 @@
+---
+
+HEADER
+
+# Concatenate all docs, stripping frontmatter
+for f in "$DOCS_DIR"/*.md "$DOCS_DIR"/golems/*.md; do
+  [ -f "$f" ] || continue
+  # Skip the llm.md file itself
+  basename=$(basename "$f")
+  [ "$basename" = "llm.md" ] && continue
+
+  echo "## $(basename "$f" .md | tr '-' ' ' | sed 's/\b\(.\)/\u\1/g')" >> "$OUTPUT"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)